etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/database/DatabasePartnerMetadataStorage.java

```diff
@@ -28,6 +28,7 @@ public class DatabasePartnerMetadataStorage implements PartnerMetadataStorage {
             new DatabasePartnerMetadataStorage();
 
     private static final String METADATA_TABLE_RECEIVED_MESSAGE_ID = "received_message_id";
+    private static final String METADATA_TABLE_SENT_MESSAGE_ID = "sent_message_id";
```

Comment:
I know this has already been discussed, so I'm sorry to bring it up again, but I think it's better not to use a constant for query column names. It makes the code less readable (we need go to the variable instantiation to know the column name) and I don't think we have the benefits. The name of the variable itself has the name of the column, so it kind of defeats the purpose of using a constant because if in the future we have to change the name of the column, we'll also have to change the name of the variable

Reply:
I don't think the we can really improve these variable names.  I'm not sure I would make them more generic.  Maybe?

It is true, this makes the SQL statements harder to read.

I think the constant solution is slightly better than having to find/replace the strings.  With the constant solution, you can rename the string once and use IDE tools to rename the constant which are more foolproof than find/replace.  _But_, it isn't that much better.

So, I feel this falls within the realm of code organization opinion.  Therefore, I say follow your heart.  Use a constant; don't use a constant.  ¯\_(ツ)_/¯

Reply:
I couldn't think of a more suitable constant name that wouldn't require being changed if the value did. I prefer having the readability of the SQL statements. I think having caution when using find/replace is worth the readability.